mappers/bluetooth_mapper/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM ubuntu:16.04
 
-CMD mkdir -p kubeedge
+RUN mkdir -p kubeedge
 
 COPY . kubeedge/
 
 WORKDIR kubeedge
 
-ENTRYPOINT ["/kubeedge/main","-logtostderr=true"]
\ No newline at end of file
+ENTRYPOINT ["/kubeedge/main","--logtostderr=true"]
```

Comment:
Do we need add compile logic here?

Reply:
As is shown in [mappers/bluetooth_mapper/Makefile](https://github.com/kubeedge/kubeedge/blob/master/mappers/bluetooth_mapper/Makefile), the bluetooth_mapper are compiled on the host.

Reply:
Another problem. [mappers/bluetooth_mapper/configuration/config.yaml](https://github.com/kubeedge/kubeedge/blob/master/mappers/bluetooth_mapper/configuration/config.yaml) are packed in the docker image, while it is a config file for cc2650-sensortag device. I think the config file should be able to be modified and mounted in the bluetooth_mapper container.